packages/react/src/patterns/sub-patterns/PictogramItem/README.md

```diff
@@ -0,0 +1,87 @@
+# Pictogram Item
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/pictogram-item';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import { Desktop } from '@carbon/pictograms-react';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { PictogramItem } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const heading = 'Lorem ipsum dolor sit amet.';
+  const copy = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+
+  const cta = {
+    type: 'text',
+    href: 'https://www.example.com',
+    copy: 'Lorem ipsum dolor',
+  };
+
+  return (
+    <PictogramItem
+      heading={heading}
+      copy={copy}
+      cta={cta}
+      Pictogram={Desktop}
+    />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the pictogram item styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name        | Required | Data Type | Default Value | Description                                                          |
+| ----------- | -------- | --------- | ------------- | -------------------------------------------------------------------- |
+| `heading`   | YES      | String    | null          | Heading text                                                         |
+| `copy`      | YES      | String    | null          | Copy text                                                            |
```

Comment:
Down here you have this all required, but in proptypes you do not.

Reply:
I've added the verification in the Proptypes